mino/minogrpc/session/mod.go

```diff
@@ -600,7 +600,7 @@ func (r *unicastRelay) Send(ctx context.Context, p router.Packet) (*ptypes.Ack,
 	ack, err := client.Forward(ctx, &ptypes.Packet{Serialized: data},
 		grpc.MaxCallRecvMsgSize(MaxMessageSize))
 	if err != nil {
-		return nil, xerrors.Errorf("client: %w", err)
+		return nil, xerrors.Errorf("client: %v", err)
```

Comment:
on the contrary, good thing that we're abstracting this.
Arguably, we should even define our own `NetworkError` type (or something like that) so that it can be exposed properly as part of the API.

Reply:
Reverted all changes from `%w` to `%v` back to `%w`. If you prefer the `%v` like the rest of the code, I re-revert it :)